docs/DataWorkflows/DataCycle.md

```diff
@@ -0,0 +1,671 @@
+
+Otherwise, the easiest way to get your CUBIC project user started is to download
+the project initialization script.  Immediately upon getting access to
+your project, log in as the project user and run:
+
+```shell
+$ wget https://raw.githubusercontent.com/PennLINC/TheWay/main/scripts/cubic/cubic-setup-project-user.sh
+$ bash cubic-setup-project-user.sh ${HOME}
+```
+
+[Download and install CuBIDS](https://bids-bond.readthedocs.io/en/latest/index.html). Note that this environment must be activated for the rest of the steps.
+
+You'll also need to install BIDS Validator:
+
+```shell
+$ conda install nodejs
+$ npm install -g bids-validator
+$ which bids-validator #to make sure it installed
+```
+
+## Stage 0: Organization
+
+The process begins when data arrives on one of our servers. While it may not
+be possible to document everything that has happened to the data prior to its
+arrival to our server, the goal of the organization step is to document as
+much as we can about the process of receiving the data, and any initial
+changes we might make to it. The person in charge of organizing a data set
+should document any changes they make in a *Data Narrative* along with any
+relevant scripts used to a git-tracked repository.
+
+Create a project root directory. Initially we suggest keeping one copy of your
+initial data that will remain untouched as a backup during early stages of
+the workflow. This is not necessary if your data is archived elsewhere online.
+In the project root create a directory structure similar to this:
+
+```
+# the actual directory structure can
+# vary depending on the needs of your project
+
+project
+├── original_data
+│   ├── sub-1
+│   ├── sub-2
+│   ├── ...
+│   └── sub-N
+└── curation
+    ├── code
+    |   ├── sandbox                # a place for untracked files
+    │   └── ProjectGithub          # the home for all your scripts, tracked by github
+    |       └── DataNarrative.md   # the data narrative file
+    └── BIDS
+```
+
+The `original_data` directory should be an unchanged copy of the original data. You can
+get a template Data Narrative file with:
+
+```shell
+$ wget https://raw.githubusercontent.com/PennLINC/RBC/master/Data_Narrative_Template.md
+```
+
+Next, add tracking to your project for scripts to be shared and tracked. 
+data:
+
+```
+project
+├── original_data
+│   ├── sub-1
+│   ├── sub-2
+│   ├── ...
+│   └── sub-N
+└── curation
+    ├── code
+    |   ├── sandbox
+    │   └── ProjectGithub         # add git here
+    |       └── DataNarrative.md
+    └── BIDS
+```
+
+You can add git like so:
+
+```bash
+$ git init
+```
+
+And remember, if you explicitly want to ignore something from tracking (with either `git` or `datalad`), add that path to a `.gitnignore` file ([see here](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) and [here](https://handbook.datalad.org/en/latest/beyond_basics/101-179-gitignore.html)).
+
+### Add NIfTI information to the sidecars
```

Comment:
Probably best to link out to CuBIDS's documentation for this, since CuBIDS _should_ have this info in one of its tutorials (if it doesn't, we should add it).

Reply:
Yep, we will have to add it